src/lib/components/Menu/Menu.js

```diff
@@ -140,16 +143,15 @@ class Menu extends Component {
       /** If there are any shared layers whose category we haven't open,
        * open them
        */
-      this.openCategoryForSharedLayers();
+      this.openCategoryForSharedLayers(sharedLayers);
     }
   }
 
   /**
    * Open category for which each of the shared layers falls under
    */
-  openCategoryForSharedLayers() {
+  openCategoryForSharedLayers(sharedLayers) {
```

Comment:
I think since now this is being used by the search and shared layers, we can rename it to a more generic name such as `openCategoryForLayers`